source/stylesheets/_layout.scss

```diff
@@ -0,0 +1,45 @@
+@import "bourbon";
+@import "settings/grid_layout";
+@import "neat";
+
+.page-header > .container {
+  @include outer-container(960px);
```

Comment:
haha, nice after all the high-end features you built in (YAGNI) you think about that? :-D